features/importers/venepaikka_harbors/tests/test_import_harbors.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import math
+
+from django.contrib.gis.geos import Point
+from django.utils.timezone import utc
+from freezegun import freeze_time
+from utils.pytest import pytest_regex
+
+from features.importers.venepaikka_harbors.importer import VenepaikkaHarborsClient
+from features.models import Feature, SourceType
+from features.tests.factories import LinkFactory
+
+HARBORS_URL = VenepaikkaHarborsClient.url
+HARBOR_ID = "SGFyYm9yTm9kZTpiNzE0ODE1NC1kYmE5LTRlM2ItOWQ2ZS1jNTYzNmEyNWFhMzk="
+
+
+def test_import_all_harbors_from_source(requests_mock, importer, harbors_response):
+    requests_mock.post(HARBORS_URL, json=harbors_response)
+
+    importer.import_features()
+
+    assert Feature.objects.count() == 2
+
+
+def test_source_type_gets_created(requests_mock, importer):
+    empty_response = {"data": {"harbors": {"edges": []}}}
+    requests_mock.post(HARBORS_URL, json=empty_response)
+
+    importer.import_features()
+
+    assert SourceType.objects.count() == 1
+    SourceType.objects.get(system="venepaikka", type="harbor")
```

Comment:
```python
assert SourceType.objects.filter(system="venepaikka", type="harbor").exists()
```
?

Reply:
I could be more explicit here that would definitely be more readable. :+1: But as a sidenote, if the get fails DoesNotExist will be raised and the test will fail.

Reply:
yep, I meant that would it be more readale with `assert ... exists()` 😅